services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/standards_standard_students_progress_report.jsx

```diff
@@ -63,30 +64,44 @@ export default class IndividualStandardsReport extends React.Component {
         accessor: 'name',
         resizable: false,
         sortType: sortTableByLastName,
-        Cell: ({row}) => (
-          <a href={row.original['student_standards_href']}><span className='row-link-disguise underlined'>{row.original['name']}</span></a>
-        )
+        Cell: ({ row }) => {
+          const { original } = row
+          const { id, name, student_standards_href } = original
+          return renderTooltipRow({ color: 'grey', icon: accountGreyIcon, id, label: name, link: student_standards_href, headerWidth: 200 })
```

Comment:
I noticed this one the headerWidth and minWidth were not the same.  This is probably correct, I just was looking for that pattern.